metadata.rb

```diff
@@ -9,8 +9,8 @@
 version          '4.0.0'
 
 depends          'acme', '~> 4.1.2'
-depends          'base'
 depends          'osl-git'
+depends          'osl-resources'
```

Comment:
I don't see where this cookbook actually needs this cookbook (only in the test cookbook)